lib/plugins/physics.js

```diff
@@ -128,7 +134,7 @@ function inject(bot) {
 
     // limit speed
     var currentMaxGroundSpeed;
-    var underBlock=bot.blockAt(pos.offset(0,-1,0));
+    var underBlock=bot.blockAt(new Vec3(pos.x,Math.ceil(pos.y-1.002),pos.z));
```

Comment:
Ah, I understand. See, if there was some global constant for this, it would be easier to understand where the 0.002 comes from and what it relates to.